kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -75,15 +83,91 @@
                 <KButton
                   :text="coreString('removeAction')"
                   appearance="flat-button"
-                  :disabled="nonCompleteStatus(download)"
                   @click="removeResource(download)"
                 />
               </td>
             </tr>
           </tbody>
         </template>
       </CoreTable>
-      <p v-if="!loading && (!downloadRequestMap || !Object.keys(downloadRequestMap).length)">
+      <!-- for small and medium screens, reorganize the table -->
```

Comment:
There's probably a way we could nest the conditionality slightly more to reduce the repetition here, but let's not change it now.